pkg/provider/azure_test.go

```diff
@@ -0,0 +1,131 @@
+package provider_test
+
+import (
+	"testing"
+
+	"github.com/Azure/azure-sdk-for-go/sdk/resourcemanager/resources/armresources"
+	"github.com/Azure/azure-sdk-for-go/services/storage/mgmt/2019-06-01/storage"
+	"github.com/pluralsh/plural/pkg/manifest"
+	"github.com/pluralsh/plural/pkg/provider"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestCreateResourceGroup(t *testing.T) {
+	resourceGroupName := "test"
+	tests := []struct {
```

Comment:
could we not define these structs inline (can be separate private structs in this file)?  Will just reduce the line lengths of these tests and improve readability a bit I think

Reply:
When we have one test scenario in the file then we can define the struct with the test parameters:
https://github.com/kubernetes/kubernetes/blob/master/pkg/kubectl/cmd/convert/convert_test.go#L31

 But for multiple test functions, it's a common pattern to use an inline struct for the test parameters: https://github.com/kubernetes/kubernetes/blob/ea0764452222146c47ec826977f49d7001b0ea8c/pkg/kubelet/kuberuntime/helpers_linux_test.go

Reply:
If it's common golang style then that's cool, just makes the func a bit longer